README.md

```diff
@@ -1,50 +1,78 @@
-![Passage Swift](https://storage.googleapis.com/passage-docs/passage-github-banner.png)
+![passage-react-native](https://storage.googleapis.com/passage-docs/github-md-assets/passage-react-native.png)
 
-[![npm version](https://badge.fury.io/js/@passageidentity%2Fpassage-react-native.svg)](https://badge.fury.io/js/@passageidentity%2Fpassage-react-native)
+[![npm version](https://badge.fury.io/js/@passageidentity%2Fpassage-react-native.svg)](https://badge.fury.io/js/@passageidentity%2Fpassage-react-native) [![React Native](https://img.shields.io/badge/React_Native-%2320232a.svg?logo=react&logoColor=%2361DAFB)](#) [![TypeScript](https://img.shields.io/badge/TypeScript-3178C6?logo=typescript&logoColor=fff)](#) ![GitHub License](https://img.shields.io/github/license/passageidentity/passage-node)
+ ![Static Badge](https://img.shields.io/badge/Built_by_1Password-grey?logo=1password)
 
-### Native passkey authentication for your React Native app
-## Welcome!
-Integrating passkey technology can be really hard. That's why we built the Passage Complete React Native SDK - to make passkey authentication easy for you and your users, all in a native Android and iOS experience.
 
-<img width="1069" alt="Passage React Native" src="https://storage.googleapis.com/passage-docs/passage_react_native.png">
+## About
 
-<br>
+[Passage by 1Password](https://1password.com/product/passage) unlocks the passwordless future with a simpler, more secure passkey authentication experience. Passage handles the complexities of the [WebAuthn API](https://blog.1password.com/what-is-webauthn/), and allows you to implement passkeys with ease. 
 
-## Installation
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
 
-```sh
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
+
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
+
+
+
+### In passage-react-native
+
+Use passage-react-native to implement Passkey Complete into your React Native app to authenticate requests and manage users.
+
+| Product | Compatible |
+| --- | --- |
+| ![Passkey Flex](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png) Passkey **Flex** | ✖️ For Passkey Flex, check out [passage-flex-react-native](https://github.com/passageidentity/passage-flex-react-native)
+| ![Passkey Complete](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png) Passkey **Complete** | ✅
+| ![Passkey Ready](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png) Passkey **Ready** | ✖️ For Passkey Ready, check out [Authentikit](https://www.npmjs.com/package/@passageidentity/authentikit)
```

Comment:
Still unclear on which Authentikit packages we should point to - I imagine for React Native pointing to the web version of Authentikit makes sense, but for native mobile READMEs we should be using Android/iOS Authentikit packages, right?

Reply:
We should point to native iOS and native android for Authentikit :)

Reply:
Sounds good, same as Flutter readme. Thanks!!